hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/Operation.java

```diff
@@ -109,6 +110,14 @@ public long getClientCallId() {
         return clientCallId;
     }
 
+    public final void setOpSentByClient(boolean value) {
```

Comment:
Can we avoid using another flag? Especially since it seems that it is used inconsistently eg someone reading this might think that all operations which are part of client message task execution have `isOpSentByClient == true` which is not the case.

Could we maybe use the existing `callerUuid` field to determine if caller is a cluster member or not? I see it is set here for all partition message tasks: https://github.com/hazelcast/hazelcast/blob/40d555dc1110da9f63fd66f504c83d4382d0384b/hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/AbstractPartitionMessageTask.java#L49